### PR TITLE
(#862) Deployment Computer Upgrade All Software

### PR DESCRIPTION
**Pull request by st3phhays**
## Description Of Changes
This documents the process of upgrading all outdated Software on a single computer. This can be done from the Computers page as well as the Computer Details page.

## Motivation and Context
This is new documentation as part of Chocolatey Central Management 0.12.0.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made
* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue
* #862

### CODE REVIEW

**Comment by st3phhays** [comment]
Images needed for this have been tracked.

**Review by gep13 — DISMISSED** [review]
LGTM!

**Comment by AdmiringWorm** [comment]
I have updated this PR to be based on #877 for temporary groups, and updated the mention of Temporary Groups to link to this section of the Groups Page.

File: input/en-us/central-management/usage/website/computers.md [hunk 1]
```diff
@@ -43,6 +43,32 @@ You will be presented with a list of the installed software packages for the mac
...
 
+## Upgrading All Outdated Software on a Computer
+
+Upgrading all outdated Software installed on a Computer can be done from two pages:
+
+1. In the leftmost column of the Computer table you will find an :gear: **Actions** menu which will display a **Upgrade Outdated Software** option.
+
+  ![Finding the Upgrade Outdated Software menu entry for a specific Computer on the Computers page](/assets/images/computers/ccm-computers-upgrade-all-software-menu.png)
+
+1. From the Computer Details page in Chocolatey Central Management, click the **Create New Deployment Plan** button and select the **Upgrade Outdated Software** option.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/computers.md:54]:
```suggestion
1. From the Computer Details page, click the :gear: **Actions** button and select the **Upgrade Outdated Software** option.
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/computers.md (hunk 1 above)
Comment (AdmiringWorm) [reply input/en-us/central-management/usage/website/computers.md]: I have updated this PR with this change.

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**